### DISCUSSION TITLE
Set a option for Chinese developers

### DISCUSSION THREAD

**Discussion by yjlc-pc** [beeware/briefcase#1740] [Ideas]
1.Set the mobile preview's URL "google.com" to "tuna.tstinghua.edu.cn".
2.Set the Gradle download URL to Huawei's Server.
3.Set the git's URL to gitee.

**Comment by mhsmith** [comment]
I don't think we're interested in playing [cat and mouse](https://en.wikipedia.org/wiki/Cat_and_mouse) with the Chinese government's censors, or verifying the integrity of mirrors that aren't under our control. My understanding is that most Chinese software developers are familar with these issues, so they can either use a VPN, or change the URLs themselves.

In particular:
* The [`briefcase new`](https://briefcase.readthedocs.io/en/stable/reference/commands/new.html) command can take an alternative template URL on the command line.
* The `briefcase create` command can take a [`template`](https://briefcase.readthedocs.io/en/latest/reference/configuration.html#template) URL from the project's configuration file.

**Comment by ColdWindScholar** [comment]
Yes , Github almost cannot connect in China
